File: src/net/airvantage/model/AirVantageException.java
package net.airvantage.model;

public class AirVantageException extends Exception {
	
	private static final long serialVersionUID = 1L;
	private AvError error;

	public AirVantageException(net.airvantage.model.AvError error) {
		super();
		this.error = error;
	}
	
	public AvError getError() {
		return error;
	}
}

File: mainActivity/src/main/java/net/airvantage/model/AirVantageException.kt
package net.airvantage.model

class AirVantageException(error: net.airvantage.model.AvError) : Exception() {
    var error: AvError? = null

    init {
        this.error = error
    }

    companion object {

        private const val serialVersionUID = 1L
    }
}
